test/test_helper.rb

```diff
@@ -1,12 +1,11 @@
 $LOAD_PATH << 'lib'
 
-require 'minitest/spec'
```

Comment:
This causes a circular dependency error:

```
/Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest.rb:987: warning: /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest.rb:987: warning: loading in progress, circular require considered harmful - /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest/test.rb
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rake-12.3.3/lib/rake/rake_test_loader.rb:5:in  `<main>'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rake-12.3.3/lib/rake/rake_test_loader.rb:5:in  `select'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rake-12.3.3/lib/rake/rake_test_loader.rb:17:in  `block in <main>'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rake-12.3.3/lib/rake/rake_test_loader.rb:17:in  `require'
	from /Users/yuki/GitHub/hyperclient/test/faraday/connection_test.rb:1:in  `<top (required)>'
	from /Users/yuki/GitHub/hyperclient/test/faraday/connection_test.rb:1:in  `require_relative'
	from /Users/yuki/GitHub/hyperclient/test/test_helper.rb:3:in  `<top (required)>'
	from /Users/yuki/GitHub/hyperclient/test/test_helper.rb:3:in  `require'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest/spec.rb:1:in  `<top (required)>'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest/spec.rb:1:in  `require'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest/test.rb:1:in  `<top (required)>'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest/test.rb:1:in  `require'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest.rb:987:in  `<top (required)>'
	from /Users/yuki/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/minitest-5.11.3/lib/minitest.rb:987:in  `require'
```

And it seems like this is unnecessary now, so just removing it.